src/acquisition/covid_hosp/facility/database.py

```diff
@@ -7,213 +7,151 @@
+        Columndef("all_adult_hospital_inpatient_bed_occupied_7_day_avg", "all_adult_hospital_inpatient_bed_occupied_7_day_avg", float),
+        Columndef("all_adult_hospital_inpatient_bed_occupied_7_day_coverage", "all_adult_hospital_inpatient_bed_occupied_7_day_coverage", int),
+        Columndef("all_adult_hospital_inpatient_bed_occupied_7_day_sum", "all_adult_hospital_inpatient_bed_occupied_7_day_sum", int),
+        Columndef("all_adult_hospital_inpatient_beds_7_day_avg", "all_adult_hospital_inpatient_beds_7_day_avg", float),
+        Columndef("all_adult_hospital_inpatient_beds_7_day_coverage", "all_adult_hospital_inpatient_beds_7_day_coverage", int),
+        Columndef("all_adult_hospital_inpatient_beds_7_day_sum", "all_adult_hospital_inpatient_beds_7_day_sum", int),
+        Columndef("ccn", "ccn", str),
+        Columndef("city", "city", str),
+        Columndef("fips_code", "fips_code", str),
+        Columndef("geocoded_hospital_address", "geocoded_hospital_address", Utils.limited_geocode),
+        Columndef("hhs_ids", "hhs_ids", str),
+        Columndef("hospital_name", "hospital_name", str),
+        Columndef("hospital_subtype", "hospital_subtype", str),
+        Columndef("icu_beds_used_7_day_avg", "icu_beds_used_7_day_avg", float),
+        Columndef("icu_beds_used_7_day_coverage", "icu_beds_used_7_day_coverage", int),
+        Columndef("icu_beds_used_7_day_sum", "icu_beds_used_7_day_sum", int),
+        Columndef("icu_patients_confirmed_influenza_7_day_avg", "icu_patients_confirmed_influenza_7_day_avg", float),
+        Columndef("icu_patients_confirmed_influenza_7_day_coverage", "icu_patients_confirmed_influenza_7_day_coverage", int),
+        Columndef("icu_patients_confirmed_influenza_7_day_sum", "icu_patients_confirmed_influenza_7_day_sum", int),
+        Columndef("inpatient_beds_7_day_avg", "inpatient_beds_7_day_avg", float),
+        Columndef("inpatient_beds_7_day_coverage", "inpatient_beds_7_day_coverage", int),
+        Columndef("inpatient_beds_7_day_sum", "inpatient_beds_7_day_sum", int),
+        Columndef("inpatient_beds_used_7_day_avg", "inpatient_beds_used_7_day_avg", float),
+        Columndef("inpatient_beds_used_7_day_coverage", "inpatient_beds_used_7_day_coverage", int),
+        Columndef("inpatient_beds_used_7_day_sum", "inpatient_beds_used_7_day_sum", int),
+        Columndef("is_corrected", "is_corrected", Utils.parse_bool),
+        Columndef("is_metro_micro", "is_metro_micro", Utils.parse_bool),
+        Columndef("previous_day_admission_adult_covid_confirmed_18-19_7_day_sum", "previous_day_admission_adult_covid_confirmed_18_19_7_day_sum", int),
+        Columndef("previous_day_admission_adult_covid_confirmed_20-29_7_day_sum", "previous_day_admission_adult_covid_confirmed_20_29_7_day_sum", int),
+        Columndef("previous_day_admission_adult_covid_confirmed_30-39_7_day_sum", "previous_day_admission_adult_covid_confirmed_30_39_7_day_sum", int),
```

Comment:
note: lots of stuff here that `black` recombined into a single line that exceeds standard line length limits. See question above for more info.